proto/stride/stakeibc/host_zone.proto

```diff
@@ -52,7 +52,7 @@ message HostZone {
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
     (gogoproto.nullable) = false
   ];
+  bool lsm_liquid_stake_enabled = 27;
   bool halted = 19;
   reserved 4, 5, 6, 7, 14, 15, 16;
-  // next id: 27
```

Comment:
Instead of removing this shouldn't you update it to say next id 28
I understand the highest id in the current proto is 27 but it could be 28 or higher, like the next id could be 29 there is no way to know just looking at the current proto the full history